calabash/Classes/FranklyServer/Routes/LPReflectionRoute.h

```diff
@@ -0,0 +1,6 @@
+#import <Foundation/Foundation.h>
+#import "LPRoute.h"
+
+@interface LPReflectionRoute : NSObject<LPRoute>
```

Comment:
I can't think of an advantage especially since none of the methods in the protocol are required.